### ISSUE TITLE
Add `--force-disabled` flag

### ISSUE THREAD

**Issue by hekaldama** [visualphoenix/jenkins-xml-to-jobdsl#8]
May be a useful feature to add a `--force-disabled` flag to runner to automatically inject a `disabled()` line into the `job` output so that people can migrate without running jobs on both servers while they verify everything is setup as desired. Could possibly implemented with in place replace post processing as well, but may be a nice feature.

**Comment by visualphoenix** [comment]
Agreed. Should be an easy enhancement for a new contributor to tackle.
